platforms/android/library/src/main/java/io/element/android/wysiwyg/inputhandlers/InterceptInputConnection.kt

```diff
@@ -333,7 +333,8 @@ internal class InterceptInputConnection(
 
     private fun replaceAll(charSequence: CharSequence) {
         editable.removeFormattingSpans()
```

Comment:
Is this still necessary now we are clearing the editable?

Reply:
Probably not, I'll double check just in case.

Reply:
It is needed, it seems like `clear()` only removes the text and selection:

<img width="391" alt="image" src="https://github.com/matrix-org/matrix-rich-text-editor/assets/480955/909e3dea-6838-467d-8e9e-5c6503dc0c98">

Here I started a list and just wrote "Let". The list item span was added 3 extra times, one for each new character.